Groups/GroupAttendanceMulti.ascx.cs

```diff
@@ -99,6 +99,24 @@ WHEN pg.Name IS NOT NULL THEN
         Order = 6,
         Key = AttributeKey.IntroLavaTemplate )]
 
+    [BooleanField( "Display LastName buttons",
+        Description = "Display a row of buttons with the first letter of LastName buttons. Default: Yes",
+        DefaultBooleanValue = true,
+        Order = 7,
+        Key = AttributeKey.DisplayLastNameButtons )]
+
+    [BooleanField( "Allow Adding Person",
+        Description = "Should block support adding new people as attendees?",
+        DefaultBooleanValue = false,
+        Order = 8,
+        Key = AttributeKey.AllowAddingPerson )]
+
+    [BooleanField( "Combine Group Attendance",
+        Description = "Should the block combine multiple group attendance to each person?",
```

Comment:
```suggestion
        Description = "Should the block display only one record per person? This will result in a group attendance for that person in each associated group.",
```